src/components/NavBar/EditorDesignName.tsx

```diff
@@ -8,15 +7,17 @@ import {
   Input,
   EditableInput,
 } from "@chakra-ui/react";
-import { useState } from "react";
-import { BiStar, BiPencil } from "react-icons/bi";
+import { BiPencil } from "react-icons/bi";
 import { useDispatch } from "react-redux";
 import { useStoreSelector } from "@/store/hooks";
+import { changeDesignName } from "@/store/reducer/courtSizeSlice";
```

Comment:
We no longer use courtSizeSlice reducer. We use courtSpecDataSlice instead. So, please move this reducer to courtSpecDataSlice.